reference/JsGetModuleHostInfo.md

```diff
@@ -0,0 +1,20 @@
+Retrieve the host info for the specified module.
+### Syntax 
+```
+CHAKRA_API
+JsGetModuleHostInfo(
+    _In_ JsModuleRecord requestModule,
+    _In_ JsModuleHostInfoKind moduleHostInfo,
+    _In_ void* hostInfo);
```

Comment:
`    _Outptr_result_maybenull_ void** hostInfo);` instead

Reply:
Now I look stupid, how on earth did I miss that..... Fixed